src/hooks/tradingView/useTradingView.ts

```diff
@@ -63,6 +69,17 @@ export const useTradingView = ({
       tvWidgetRef.current = tvChartWidget;
 
       tvWidgetRef.current.onChartReady(() => {
+        tvWidgetRef?.current?.headerReady().then(() => {
+          displayButtonRef.current = tvWidgetRef?.current?.createButton();
+          displayButtonRef.current.innerHTML = `<span>${stringGetter({
+            key: STRING_KEYS.ORDER_LINES,
```

Comment:
since we are using `stringGetter` in this hook, we should add it to dep array so the UI will update this `button` and `title` on language change.

Reply:
hmm I think right now, on language change, the whole chart gets destroyed and re-mounted (with the updated language) because of `selectedLocale` being a [part](https://github.com/dydxprotocol/v4-web/pull/287/files/38dd5f6e88c41ff4102134393e0764dabfeab7a6#diff-c8b3ff077660b9f0fa861c122ef4a719a9d312fd37ac918cdadca357f568f285L83) of the dep array. Would it still be better to explicitly add something like `displayButtonText` to the dep array?

Reply:
ah gotcha. selectedLocale works